cpp/MODULE.bazel

```diff
@@ -0,0 +1,12 @@
+module(name = "maplibre-tile-spec")
```

Comment:
Should be moved to the root I think.

Reply:
My longer-term goal was to use `just` for the cpp build as well, and allow Bazel to build it from `vendor`, but I ended up needing to add this to satsfy Bazel.  Possibly I missed something and it's not really necessary?